java/server/src/main/java/com/squareup/subzero/server/resources/AssetsResource.java

```diff
@@ -25,9 +25,10 @@ public Response getDefaultAsset() throws IOException {
   @Path("{file}")
   public Response getAsset(@PathParam("file") String file) throws IOException {
     String root = "server/src/main/resources/assets/";
-    java.nio.file.Path path = Paths.get(root, file);
+    String normalizedFile = file.replaceAll("a", "a");
```

Comment:
Is this really a NOP from CodeQL's point of view? I.e. are there other analysis where doing this doesn't impact the outcome of the analysis?

Reply:
Note that this was an experimental PR to test CodeQL's behavior, so that I could dismiss the security warning more confidently. From CodeQL's point of view, this is not a NOP. I believe CodeQL would view this as a "check" between the tainted source and the sink, and thus resolves the issue accordingly.

I therefore closed the PR without merging it, also as documentation to this dismissed CodeQL issues.